sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/IntervalUtils.scala

```diff
@@ -57,6 +57,11 @@ object IntervalUtils {
   }
   import IntervalUnit._
 
+  private val MAX_DAY = 106751991L
+  private val MAX_HOUR = 2562047788L
+  private val MAX_MINUTE = 153722867280L
+  private val MAX_SECOND = 9223372036854L
```

Comment:
This is not used in the code. Found the place where the max seconds is calculated:
https://github.com/apache/spark/blob/3953754f36656e1a0bee16b89fae0142f172a91a/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/IntervalUtils.scala#L635
Could you replace such places in the file (maybe other places), and put a formula here instead of numbers.

Reply:
Done